app/services/return_questionnaire_results_hash.rb

```diff
@@ -0,0 +1,17 @@
+class ReturnQuestionnaireResultsHash
```

Comment:
As I understand it this data's only going to be used for printing the results of the survey, hence the simple JSON blob. Would it be more readable if we used the actual questions we ask as the keys (i18n values may change in the future), and for the values, munge together the answer and the explanation? eg

```
{
  "Please rate your experience of giving a reference" => "Good. I liked it",
  "Please rate our guidance" => "Not bad."
}
```

Reply:
It may not be a requirement today but if, in future, somebody ever asked for say the % of respondents that said the experience was good or better then this could make it a bit more tricky (due to those i18n keys being volatile).

Reply:
This is interesting. Perhaps we could have a quick chat after stand up tomorrow!

Reply:
I wasn't there — let's talk today!